docs/reference/connector/apis/update-connector-last-sync-api.asciidoc

```diff
@@ -0,0 +1,134 @@
+[[update-connector-last-sync-api]]
+=== Update connector last sync stats API
+
+preview::[]
+
+++++
+<titleabbrev>Update connector last sync stats</titleabbrev>
+++++
+
+Updates the fields related to the last sync of a connector.
+
+This action is used for analytics and monitoring.
+
+[[update-connector-last-sync-api-request]]
+==== {api-request-title}
+
+`PUT _connector/<connector_id>/_last_sync`
+
+[[update-connector-last-sync-api-prereq]]
+==== {api-prereq-title}
+
+* To sync data using connectors, it's essential to have the Elastic connectors service running.
+* The `connector_id` parameter should reference an existing connector.
+
+[[update-connector-last-sync-api-path-params]]
+==== {api-path-parms-title}
+
+`<connector_id>`::
+(Required, string)
+
+[role="child_attributes"]
+[[update-connector-last-sync-api-request-body]]
+==== {api-request-body-title}
+
+`last_access_control_sync_error`::
+(Optional, string) The last error message related to access control sync, if any.
+
+`last_access_control_sync_scheduled_at`::
+(Optional, datetime) The datetime indicating when the last access control sync was scheduled.
+
+`last_access_control_sync_status`::
+(Optional, ConnectorSyncStatus) The status of the last access control sync.
+
+`last_deleted_document_count`::
+(Optional, long) The number of documents deleted in the last sync process.
+
+`last_incremental_sync_scheduled_at`::
+(Optional, datetime) The datetime when the last incremental sync was scheduled.
+
+`last_indexed_document_count`::
+(Optional, long) The number of documents indexed in the last sync.
+
+`last_sync_error`::
+(Optional, string) The last error message encountered during a sync process, if any.
+
+`last_sync_scheduled_at`::
+(Optional, datetime) The datetime when the last sync was scheduled.
+
+`last_sync_status`::
+(Optional, ConnectorSyncStatus) The status of the last sync.
+
+`last_synced`::
+(Optional, datetime) The datetime of the last successful synchronization.
+
+
+The value of `ConnectorSyncStatus` is one of the following lowercase strings representing different sync states:
+  - `canceling`: The sync process is in the process of being canceled.
+  - `canceled`: The sync process has been canceled.
+  - `completed`: The sync process completed successfully.
+  - `error`: An error occurred during the sync process.
+  - `in_progress`: The sync process is currently underway.
+  - `pending`: The sync is pending and has not yet started.
+  - `suspended`: The sync process has been temporarily suspended.
```

Comment:
```suggestion

* `canceling`: The sync process is in the process of being canceled.
* `canceled`: The sync process has been canceled.
* `completed`: The sync process completed successfully.
* `error`: An error occurred during the sync process.
* `in_progress`: The sync process is currently underway.
* `pending`: The sync is pending and has not yet started.
* `suspended`: The sync process has been temporarily suspended.
```

Reply:
Updates to asciidoc syntax

Reply:
Oh I'm still on a `.md` team it seems, thank you!

Reply:
I do this at least once a day :)